packages/core/src/view/geometry/node/StencilShape.ts

```diff
@@ -36,6 +36,25 @@ import AbstractCanvas2D from '../../canvas/AbstractCanvas2D';
 import { AlignValue, ColorValue, VAlignValue } from '../../../types';
 import { getNumber } from '../../../util/StringUtils';
 
+/**
+ * Configure global settings for stencil shapes.
+ * @experimental subject to change or removal
+ */
+export const StencilShapeGlobalSettings = {
```

Comment:
**thought:** why not just naming it `StencilShapeGlobals`? if we rename it, update the documentation and the PR description.